### PR TITLE
CSS Changes and Chart Updates

### PR DESCRIPTION
**Pull request by jonathan-best**
* Allow filtering of specific items when using min doc count
* Reduce the screen width (validated size with claire)
* Up the z-index of loading text and add min height to charts to stop popping effect
* Add component class for Mo as a test handle

### CODE REVIEW

**Comment by peter-mouland** [comment]
cool - just taking a look now...

**Comment by peter-mouland** [comment]
could we play with the white space in the `modifiers section too?

i.e.
 * reduce the margin at the bottom of the nav (to 15px)
 * reduce the vertical gap between form controls (to 6px)
 * increase the horizontal gap between tags (to 15px)
 * move the main content/headline-stats down (10px)

screen shot 2015-10-14 at 1 57 15 pm

**Comment by peter-mouland** [comment]
desired???

screen shot 2015-10-14 at 2 04 09 pm

File: src/server/formatters/Articles.js [hunk 1]
```diff
@@ -65,6 +66,17 @@ export default function formatData(data) {
...
+function formatAuthors(authors) {
+  if (authors.length) {
+    return 'Anonymous';
+  }
+
+  let lastAuthor = authors.pop();
+  if (!authors.length) {
+    return lastAuthor;
+  }
+  return authors.join(", ") + ' and ' + lastAuthor;
```
Comment (peter-mouland) [inline src/server/formatters/Articles.js]: whoops rebase error - was moved to `formatAuthors.js`

File: src/server/formatters/Articles.js [hunk 2]
```diff
@@ -56,6 +56,7 @@ export default function formatData(data) {
         }
       };
 
+      console.log(results.article.regions)
```
Comment (peter-mouland) [inline src/server/formatters/Articles.js]: whoops

Comment (jonathan-best) [inline src/server/formatters/Articles.js]: Done :+1:

File: src/server/formatters/Articles.js (hunk 1 above)
Comment (jonathan-best) [inline src/server/formatters/Articles.js]: Done :+1:

**Comment by peter-mouland** [comment]
cool :dancer:

**Pull request merged**